.github/workflows/integration-tests.yaml

```diff
@@ -116,9 +131,27 @@ jobs:
             --set gitea.admin.password=${GITEA_ADMIN_PASSWORD} \
             --wait
 
+      - name: Check Helm repo for image
+        id: helm_repo_check
+        shell: bash
+        env:
+          KEPTN_VERSION: ${{ matrix.keptn-version }}
+        run: |
+          echo "Keptn Version: $KEPTN_VERSION"
+          helm repo add keptn https://charts.keptn.sh
+          helm_search=$(helm search repo keptn/keptn --version $KEPTN_VERSION)
+          
+          if [[ "$helm_search" != "No results found" ]]; then
```

Comment:
Please do not use freetext output for testing, Try using `-o json` along with `jq` to test how many elements are returned.

Reply:
Sounds like a more stable solution. I just changed it.